params.yml

```diff
@@ -31,6 +31,9 @@ args:
   # - 'zkevm': Use the legacy sequencer (https://github.com/0xPolygonHermez/zkevm-node).
   sequencer_type: erigon
 
+  # Deploy agglayer.
+  deploy_agglayer: true
```

Comment:
nit: I agree with the fact that we need to decouple the agglayer and the central/bridge environment deployments. Maybe this could be defined close to the other `deploy_` params.